pkg/systemd/quadlet/quadlet.go

```diff
@@ -268,6 +269,18 @@ func usernsOpts(kind string, opts []string) string {
 	return res.String()
 }
 
+func parseUserGroup(in string) string {
+	if strings.HasPrefix(in, "%") {
+		return in
+	}
```

Comment:
please use the already existing function `startsWithSystemdSpecifier()`

Reply:
done